### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by kgscialdone** [KatrinaKitten/facet#3] [Announcements]
This is the primary discussion board for Facet. Feel free to ask for help or discuss projects, ideas, feature suggestions, or even just what you're working on that uses Facet.

You can also find Facet discussions in its channel on the [HTMX Discord server](https://discord.gg/fmqVtPABQh).

## Rules
- Don't be a jerk.
- Use your common sense.
- Stay on-topic and respectful of other users.
- Self-promotion for projects using Facet is allowed, otherwise please don't.

**Comment by airbr** [comment]
Hi everyone! I made this with an earlier version of Facet: https://facet-workspace.netlify.app/ just wanted to share :)
